### PR TITLE
rough intro to bluebird with generators

### PR DESCRIPTION
**Pull request by cultofmetatron**
added a brief overview of generators with information on how to use them with bluebird.

### CODE REVIEW

**Comment by gdi2290** [comment]
:+1:

**Comment by petkaantonov** [comment]
The unfinished article tag shouldn't be removed yet. The article is also supposed to explain why generators don't really deprecate promises right now, which is about all the aggregate methods that cannot be replicated, the sucky syntactic catch statement, lack of `using` statement, no long stack traces etc.

File: docs/docs/what-about-generators.md [hunk 1]
```diff
@@ -3,4 +3,63 @@ id: what-about-generators
 title: What About Generators?
 ---
 
-[what-about-generators](unfinished-article)
+[← Back To API Reference](/docs/api-reference.html)
```
Comment (petkaantonov) [inline docs/docs/what-about-generators.md]: You can link to the actual api documentation by writing

`[Promise.coroutine](.)`

**Comment by benjamingr** [comment]
@cultofmetatron are you still interested in finishing this?

**Pull request closed without merging**